Sources/AccessibilitySnapshot/Core/BUILD.bazel

```diff
@@ -0,0 +1,73 @@
+load(
+    "@build_bazel_rules_apple//apple:ios.bzl",
+    "ios_framework",
+)
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library",
+)
+
+# -- AccessibilitySnapshotCore
+
+swift_library(
+    name = "AccessibilitySnapshotCore.lib",
+    srcs = glob(
+        ["Swift/**/*.swift"],
+        allow_empty = False,
+    ),
+    data = glob(
+        ["Swift/Assets/**/*"],
+        allow_empty = True,
+    ),
+    defines = ["BAZEL_PACKAGE"],
+    module_name = "AccessibilitySnapshotCore",
+    tags = ["manual"],
+    visibility = ["//visibility:public"],
+    deps = [],
```

Comment:
Doesn't this need to depend on `"AccessibilitySnapshotCore_ObjC.lib"`?

Reply:
Even though there's no direct code usage, the Swift lib won't work correctly unless `+[ASAccessibilityEnabler load]` is run.

Reply:
Fixed this one too